packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
+
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+export let title = '';
+export let message = '';
+export let variant: Variants = 'info';
+export let progress = 1;
+export let exitable = false;
```

Comment:
Could you please add docstrings to all props? [Example](https://github.com/viamrobotics/prime/blob/main/packages/core/src/lib/button.svelte#L15-L39)